### ISSUE TITLE
Next feature release

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#4953]
While we could immediately do a release from `release` branch also our `master` branch does not look that bad IMHO.

Currently I do only see some needed improvements for the Ignore list feature, which have been discussed as mandatory before releasing this feature:

- [x] #4712  <-- no longer relevant for `release`branch as this feature was disabled
- [x] #4711 <-- no longer relevant for `release`branch as this feature was disabled

Found issues while testing the RC:
- [x] #4985 
- [x] #4987 
- [x] #4988

- [x] #4806  <-- To be checked whether this might be critical to fix before release

Another not critical possible regression:
- [x] #4986 <-- seems not to be a regression but something else. Marked as done for the release. See issue for details.

Furthermore we have some still unclear regression issues which I am about to verify/analyze:
- [x] #4827 
- [x] #4834 

Change of links to website:
- [x] #4954 

Feedback?
Bugfix release or tackle those issues and target a feature release quite soon?

**Comment by Lineflyer** [comment]
One more thing:
As we are hopefully about to release the new webpage we should more or less try to sync our next release with that as we have to change some FAQ URLs in our sources to keep them working.
Refer to https://github.com/cgeo/cgeo.github.io/issues/27 for more details.

**Comment by rsudev** [comment]
I'd rather try to release from master. What comes to my mind additionally is the question, whether we should try to change the back-button behaviour in EditWaypoint (there was a discussion to not save automatically, but to ask whether to discard changes. I just cannot find that right now)

**Comment by Lineflyer** [comment]
@rsudev How was it before that change? AFAIR we did also save on back button before. In that case it would not be such a problem...but if the bahviour changed we should tackle this.

**Comment by kumy** [comment]
#4827 should be solved.

**Comment by Lineflyer** [comment]
@kumy 
Just a small remark regarding our typical working mode when fixing issues:
- Assign yourself to that issue to make more visible that you are the one who fixed it.
- Once it is merged to `release` or `master` please set the correpsonding milestone for that issue (Release branch or Master branch). Doing so allows us to have a list of all fixed issues on a branch.

I just did that for #4827 as an example.

**Comment by kumy** [comment]
@Lineflyer thanks for the clarification, I didn't know about milestone stuff

**Comment by rsudev** [comment]
From IRC discussion:

I think we should take this week and try to trim down the risk level of our changes - and stabilze resources at the same time, so we can call a last round for translation

**Comment by Lineflyer** [comment]
@Bananeweizen Are you able to look into #4711 and #4712 next days? For the last release we excluded the ignore list feature because we agreed on IRC that we need those two things as "security measure" before releasing it.

**Comment by Lineflyer** [comment]
PR #4975 has been prepared for #4954

**Comment by Lineflyer** [comment]
As written on IRC I am fine with moving master to release as it would be stable enough to do a quick release right away.
But open in general:
- Issues listed above
- Pending PRs for bugfixes?!

It would be the right time to take the website live before releasing btw.

**Comment by Lineflyer** [comment]
Call for translations sent on Crowdin (as I am pretty sure strings are complete for now).

**Comment by samueltardieu** [comment]
Master has been moved to release and changelogs have been merged.

**Comment by Lineflyer** [comment]
Added #4985 above

**Comment by Lineflyer** [comment]
@kumy: 
I just tested the new release candidate with respect to initial behaviour of the geokrety connector.
Right now AFAICS a "standard" user will still have requests to geokretymap.org enabled, so if he opens a GC cache the GK inventory will be added although he did not actively select anything.
I am well fine with that (because it makes users aware of the new feature and GK in general), I just want to make sure that the possible load is OK with you and your server?!

**Comment by Lineflyer** [comment]
Added #4986

**Comment by kumy** [comment]
@Lineflyer GKM server is ready and on a dedicated DigitalOcean instance since 1 month. We've benched the API service with `siege`. We can handle ~500 simultaneous connexions.
I stay ready to pop a new instance if needed.

Will the "deployment" be progressive? or all users will see the update simultaneously?

BTW: project is there: https://github.com/GeoKretyMap :)

**Comment by Bananeweizen** [comment]
ignoring caches is now disabled on release and has a confirmation on master. I'm off for this week.

**Comment by Lineflyer** [comment]
Thanks @Bananeweizen . I set this issues on done here for the next release.

**Comment by Lineflyer** [comment]
@rsudev  
Any chance you might look into #4985 regarding back key behaviour in waypoint creation?
I talked with some users who tested nightlies and/or RC and they all prefer another solution here.

**Comment by rsudev** [comment]
What can easily be done is not to save on back. The warning (in case of unsaved changes) might be more tricky.
Unfortunately am I more or less unavailable for the next days (will be on vacation from Thursday to Sunday), so from my side I can probably not look into it before early next week.

**Comment by Lineflyer** [comment]
The RC version looks rather stable in my tests on the long weekend.
I had one crash after long time of using the live map, but was unable to debug/reproduce. Most probably a problem we have had already in the past.

However before releasing this version I am still a little concerned about the change of functionality in the waypoint edit view (pressing back key will save changes but discarded them in the past).
Any chance we at least get the workaround mentioned by @rsudev above in place for this release (i.e. no save on back). Additionally @konpapa seemed to start working on a dialog but I am not sure if it is wise to implement it in this release as well (assuming it is ready soon)? Comments?

**Comment by Lineflyer** [comment]
Added #4806 above.

**Comment by Lineflyer** [comment]
According to the list of issues in the iniitial post we are done.
I will now publish a link to the RC on FB, I suggest to wait for some feedback there and then release next days (and maybe take some translations if there are new ones).

**Comment by Lineflyer** [comment]
Created a new RC 2015.06.14 including the fix for #5030

**Comment by Lineflyer** [comment]
Version 2015.06.15 was just published to 1% of the market.

@kumy: Just a heads up in case you want to watch your server.

**Comment by kumy** [comment]
@Lineflyer Thanks, I'll be attentive :)

**Comment by Lineflyer** [comment]
Already increased to 5% as 1% did not show anything on GooglePlay or status.cgeo.org although app was available on Play.

**Comment by kumy** [comment]
Everything OK until now... increase to 10%?

**Comment by Lineflyer** [comment]
I will wait some more hours to check if any crash reports come in on GooglePlay and then go to 20% if this is OK for you @kumy 
I am also available on IRC.

**Comment by kumy** [comment]
Not at home right now. But ok :)

**Comment by Lineflyer** [comment]
Set to 50% right now as agreed on IRC:

**Comment by kumy** [comment]
Everything OK on GKM until now... I still keep an eye on ;)

**Comment by Lineflyer** [comment]
Set to 100% now as agreed on IRC.

Notification will be send tonight, to give GooglePlay some time.

**Comment by Lineflyer** [comment]
Feature release is complete.
